policy/resolver.go

```diff
@@ -273,7 +273,6 @@ func (s *LocalServices) CreatePolicyObject(policyMrn string, ownerMrn string) *P
 			Policies: []*PolicyRef{},
 			Checks:   []*explorer.Mquery{},
 			Queries:  []*explorer.Mquery{},
-			Filters:  &explorer.Filters{},
```

Comment:
fyi: when creating filters either create the whole thing (with items map) or none at all; here we auto-generate the first group, but without any filters. That should be expected and to be handled downstream, since groups may not have filters (since they can be parsed this way from bundles as well)